src/java/org/apache/cassandra/db/compaction/unified/Controller.java

```diff
@@ -622,9 +628,9 @@ public static Map<String, String> validateOptions(Map<String, String> options) t
                 if (sizeInBytes < 0)
                     throw new ConfigurationException(String.format("Invalid configuration, %s should be greater than or equal to 0 (zero)",
                                                                    MIN_SSTABLE_SIZE_OPTION));
-                int limit = (int) Math.ceil(targetSSTableSize * INVERSE_SQRT_2);
+                long limit = (long) Math.ceil(targetSSTableSize * INVERSE_SQRT_2);
                 if (sizeInBytes >= limit)
-                    throw new ConfigurationException(String.format("Invalid configuration, %s (%s) should be less than the target size minimum: %s",
```

Comment:
We report 2 variables to the user

1) `limit` - this is computed from `targetSSTableSize`; `targetSSTableSize` is what the user gave
2)`sizeInBytes` - this is what the user gave

Is there a reason for not showing the user `targetSSTableSize`?  I am told that my configs don't match, but I don't know what 2 configs; i only know one of them